src/site/content/es-es/vitals/index.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: page
+title: Web Vitals
+subhead: Métricas esenciales para un sitio saludable
+description: Métricas esenciales para un sitio saludable
+authors:
+  - philipwalton
+date: 30-04-2020
+updated: 21-07-2020
+masthead: web-vitals.svg
+tags:
```

Comment:
I think we should translate tags

- métricas
- rendimiento
- web-vitals

Reply:
oh I don't think we can translate them at this level as they will affect the urls. But we might be able to translate how they're displayed on the site.